doc/reference/image_format.md

```diff
@@ -1,72 +1,75 @@
 (image-format)=
 # Image format
 
-## Image format
+Images contain a root file system and a metadata file that describes the image.
+They can also contain templates for creating files inside an instance that uses the image.
 
-LXD currently supports two LXD-specific image formats.
+Images can be packaged as either a unified image (single file) or a split image (two files).
 
-The first is a unified tarball, where a single tarball
-contains both the instance root and the needed metadata.
+## Content
 
-The second is a split model, using two files instead, one containing
-the root, the other containing the metadata.
+Images for containers have the following folder structure:
 
-The former is what's produced by LXD itself and what people should be
-using for LXD-specific images.
-
-The latter is designed to allow for easy image building from existing
-non-LXD rootfs tarballs already available today.
-
-### Unified tarball
+```
+metadata.yaml
+rootfs/
+templates/
+```
 
-Tarball, can be compressed and contains:
+Images for VMs have the following folder structure:
 
-- `rootfs/`
-- `metadata.yaml`
-- `templates/` (optional)
+```
+metadata.yaml
+rootfs.img
+templates/
+```
 
-In this mode, the image identifier is the SHA-256 of the tarball.
+For both instance types, the `templates/` folder is optional.
 
-### Split tarballs
+### Metadata
 
-Two (possibly compressed) tarballs. One for metadata, one for the rootfs.
+The `metadata.yaml` file contains information that is relevant to running the image in LXD.
+It includes the following information:
 
-`metadata.tar` contains:
+```yaml
+architecture: x86_64
+creation_date: 1424284563
+properties:
+  description: Ubuntu 22.04 LTS Intel 64bit
+  os: Ubuntu
+  release: jammy 22.04
+templates:
+  ...
+```
 
-- `metadata.yaml`
-- `templates/` (optional)
+The `architecture` and `creation_date` fields are mandatory.
+The `properties` field contains a set of default properties for the image.
+The `os`, `release`, `name` and `description` fields are commonly used, but are not mandatory.
 
-`rootfs.tar` contains a Linux root file system at its root.
+The `templates` field is optional.
+See {ref}`image_format_templates` for information on how to configure templates.
 
-In this mode the image identifier is the SHA-256 of the concatenation of
-the metadata and rootfs tarball (in that order).
+### Root file system
 
-### Supported compression
+For containers, the `rootfs/` directory contains a full file system tree of the root directory (`/`) in the container.
 
-LXD supports a wide variety of compression algorithms for tarballs
-though for compatibility purposes, `gzip` or `xz` should be preferred.
+Virtual machines use a `rootfs.img` file instead of a `rootfs/` directory.
```

Comment:
Can you say "Virtual machines use a `rootfs.img` qcow2 file instead..." 
I know you explained it further down but would be useful to briefly mention it here too.